ora2/jobs/SetOra2VersionOnSandbox.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    JOB_FOLDER_NAME: some-folder
+    CONTACT_EMAIL : e-mail-contact@example.com
+
+    The following parameters are needed for the Runansible Job below
+
+    SSH_AGENT_KEY: dictionary of key per environment
+    CONFIGURATION_REPO: standard default available
+    CONFIGURATION_BRANCH: defaults to master
+    ACCESS_CONTROL: list of Jenkins matrix targets who should get read access (edx*team or username)
+    SCHEDULE: Cron syntax consumed by Jenkins triggered builds
+    SSH_USER: ansible connect user - defaults to ubuntu.  Make sure this matches up with SSH_AGENT_KEY
+    CONTACT_EMAIL: email address to be notified when this job fails
+    PLAYBOOK:  The playbook used in this job
+*/
+
+package ora2.jobs
+
+import static devops.jobs.RunRemoteAnsiblePlaybook.job as RunRemoteAnsiblePlaybookJob
+class SetOra2VersionOnSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        def jobRoot = extraVars.get("JOB_FOLDER_NAME")
+
+        def jobName = 'SetOra2VersionOnSandbox'
+        if (jobRoot != null && jobRoot != '') {
+            jobName = jobRoot + '/SetOra2VersionOnSandbox'
+        }
+        return dslFactory.job(jobName) {
+
+            parameters {
+
+                stringParam('ORA2_VERSION',
+                            'master',
+                            'The version/branch name of ORA2 to fetch/checkout on target machine'
+                            )
+                stringParam("SANDBOX_HOST",
+                            "staubina-ora2.sandbox.edx.org",
+                            "The host name and address of the Sandbox to be updated"
+                           )
+                stringParam("NOTIFY_ON_FAILURE",
+                            "astaubin@edx.org",
+                            "Email to notify on failure")
+            }
+
+            concurrentBuild(false)
+            def ansibleJobName = 'ansibleOraVersion'
+            if (jobRoot != null && jobRoot != '') {
+                ansibleJobName = jobRoot + '/ansibleOraVersion'
+            }
+            RunRemoteAnsiblePlaybookJob(dslFactory, ansibleJobName, 'development', 'edx', extraVars + ['ORA2_VERSION':'$ORA2_VERSION', 'SANDBOX_HOST':'$SANDBOX_HOST'] )
```

Comment:
This looks like you're set up to pass down the parameters you want, so I'm confused why the ORA2 things were in RunRemoteAnsiblePlaybookJob